src/ds/d400/d400-color.cpp

```diff
@@ -126,7 +126,7 @@ namespace librealsense
     {
         firmware_version fw_ver = firmware_version( get_info( RS2_CAMERA_INFO_FIRMWARE_VERSION ) );
 
-        if( fw_ver >= firmware_version( 5, 10, 9, 0 ) )
+        if( fw_ver >= firmware_version( 5, 10, 9, 0 ) && _pid != ds::RS405_PID ) //RS405 does not support an actuall RGB sensor
```

Comment:
Typo `actual`, also call it D405, not RS405

Reply:
Does not have an actual color sensor?

Reply:
So as I understand you are saying its OK thats it doesnt work and we wish to block it?

Reply:
D405 does not have an RGB sensor, color stream from IR sensor only. FW uses a different opcode for RGB/depth sensor. The way it was implemented (since D405 have only one sensor) this register will override depth sensor option and replace it with the wrong opcode. The fix here is not to register the option a second time.

Reply:
Got it thanks

Reply:
You can drop my review and use @OhadMeir only :)

Reply:
Done